transport.go

```diff
@@ -134,3 +135,20 @@ func (t *Transport) refreshToken() error {
 
 	return nil
 }
+
+func addAcceptHeader(headers http.Header) {
```

Comment:
It would be nice to document what the function does. Perhaps something like this:

```Go
// addAcceptHeader adds acceptHeader to headers, but only
// if the current "Accept" header is not set, or already accepts JSON.
func addAcceptHeader(headers http.Header) {
```

Reply:
Copied it directly (seemed like a good description) and pushed.